testing/sdk_utils.py

```diff
@@ -44,4 +44,12 @@ def try_throws_n_times(predicate, attempts=1, sleep_seconds=0):
                 raise
             out("Encountered an exception: {}".format(traceback.format_exc()))
 
-        time.sleep(sleep_seconds)
\ No newline at end of file
+        time.sleep(sleep_seconds)
+
+def get_foldered_name(service_name):
+    # DCOS 1.9 & earlier don't support "foldered", service names aka marathon
+    # group names
+    if shakedown.dcos_version_less_than("1.10"):
+        return "test__integration__" + service_name
```

Comment:
double underscores in service names should fail* regardless of dcos version. should probably just use single underscores

* https://github.com/mesosphere/dcos-commons/blob/master/sdk/scheduler/src/main/java/com/mesosphere/sdk/config/validate/ServiceNameCannotContainDoubleUnderscores.java